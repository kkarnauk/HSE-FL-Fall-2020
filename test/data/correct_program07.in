f:-f.


f :- f, (f, f, f); (f; (f; (f, g, f (ff) (ff)))).

f:-fdsdsdsdssd, ddfd (
sdasd (sads )
).
