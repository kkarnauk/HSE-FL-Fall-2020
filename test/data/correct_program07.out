|relation
|	|head
|	|	|identifier = 'f'
|	|body
|	|	|identifier = 'f'

|relation
|	|head
|	|	|identifier = 'f'
|	|body
|	|	|disjunction
|	|	|	|conjunction
|	|	|	|	|identifier = 'f'
|	|	|	|	|conjunction
|	|	|	|	|	|identifier = 'f'
|	|	|	|	|	|conjunction
|	|	|	|	|	|	|identifier = 'f'
|	|	|	|	|	|	|identifier = 'f'
|	|	|	|disjunction
|	|	|	|	|identifier = 'f'
|	|	|	|	|disjunction
|	|	|	|	|	|identifier = 'f'
|	|	|	|	|	|conjunction
|	|	|	|	|	|	|identifier = 'f'
|	|	|	|	|	|	|conjunction
|	|	|	|	|	|	|	|identifier = 'g'
|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|identifier = 'f'
|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|identifier = 'ff'
|	|	|	|	|	|	|	|	|atom
|	|	|	|	|	|	|	|	|	|identifier = 'ff'

|relation
|	|head
|	|	|identifier = 'f'
|	|body
|	|	|conjunction
|	|	|	|identifier = 'fdsdsdsdssd'
|	|	|	|atom
|	|	|	|	|identifier = 'ddfd'
|	|	|	|	|atom
|	|	|	|	|	|identifier = 'sdasd'
|	|	|	|	|	|atom
|	|	|	|	|	|	|identifier = 'sads'